src/Traits/GenericActionsTrait.php

```diff
@@ -54,9 +54,16 @@ abstract public function paginate($object = null, array $settings = []);
      */
     protected function loadFilteredChildren(string $id): array
     {
+        $folder = $this->Objects->loadObject($id, 'folders');
+        $order = $folder['custom_props'] && $folder['custom_props']['children_order'] ?? $folder['custom_props']['children_order'];
```

Comment:
Not sure about this assignment. Also `children_order` is not a custom property.
```suggestion
        $order = $folder['children_order'] ?? null;
```

Reply:
Actually children_order is a custom property when we first fetch data from the DB:

<img width="285" alt="image" src="https://user-images.githubusercontent.com/22395670/209312493-3fc5ca48-fecf-4a23-a2f1-44588b8b5663.png">

I also tested and the above code is working as expected.